Quotient/connection.cpp

```diff
@@ -326,31 +421,32 @@ void Connection::Private::completeSetup(const QString& mxId, bool newLogin,
         q->loadCapabilities();
         q->user()->load(); // Load the local user's profile
     }
+    auto doCompleteSetup = [this, mxId, deviceId, accessToken](){
```

Comment:
```suggestion
    auto doCompleteSetup = [this, mxId, deviceId, accessToken](QKeychain::Job* = nullptr){
```
I'm afk and can't check it but that should make the lambda conform to `QFuture::then` interface below.